libs/ui-toolkit/src/components/ag-grid/ag-grid-dynamic-themed.tsx

```diff
@@ -35,10 +35,12 @@ export const AgGridThemed = ({
       style={style}
     >
       {theme === 'dark' ? (
+        // @ts-ignore TFE import
```

Comment:
chore (blocking): Should be able to remove these ts-ignores

Reply:
When removed, I get the following:

`TS2786: 'AgGridDarkTheme' cannot be used as a JSX component.   Its element type 'ReactElement<any, any> | Component<{ children: ReactElement<any, string | JSXElementConstructor<any>>; }, any, any> | null' is not a valid JSX element.     Type 'Component<{ children: ReactElement<any, string | JSXElementConstructor<any>>; }, any, any>' is not assignable to type 'Element | ElementClass | null'.       Type 'Component<{ children: ReactElement<any, string | JSXElementConstructor<any>>; }, any, any>' is not assignable to type 'ElementClass'.         The types returned by 'render()' are incompatible between these types.           Type 'React.ReactNode' is not assignable to type 'import("/Users/samkeen/Documents/Vega/frontend-monorepo/node_modules/@types/react-dom/node_modules/@types/react/index").ReactNode'.             Type '{}' is not assignable to type 'ReactNode'.               Type '{}' is missing the following properties from type 'ReactPortal': key, children, type, props`

Reply:
I've kept these in for now, to revisit once the branch is merged. Willing to remove and fix if deemed necessary pre-merge though.